### PR TITLE
CDSR-437: log correlation id and correct batch count

### PR DESCRIPTION
**Pull request by mikail-khan**

### CODE REVIEW

File: app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala [hunk 1]
```diff
@@ -50,12 +51,13 @@ class DefaultCcsConnector @Inject() (http: HttpClient, val config: ServicesConfi
...
     hc: HeaderCarrier
   ): EitherT[Future, Error, HttpResponse] = {
+
     logger.info(
-      "submitToCcs request CORRELATION_ID: " + ccsSubmissionPayload.headers
-        .find(_._1 === CustomHeaderNames.X_CORRELATION_ID)
-        .map(_._2)
-        .getOrElse("CORRELATION_ID was not found")
+      s"Ccs submission request correlation id: ${extraHeaders.extraHeaders
+        .find(p => p._1 === CustomHeaderNames.X_CORRELATION_ID)
```
Comment (gabor-dorcsinecz) [inline app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala:57]: Whats wrong with _._1 ?

File: app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala [hunk 2]
```diff
@@ -50,12 +51,13 @@ class DefaultCcsConnector @Inject() (http: HttpClient, val config: ServicesConfi
...
+        .fold("No correlation id found")(c => c._2)}"
```
Comment (gabor-dorcsinecz) [inline app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala:58]: and _._2

File: app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala (hunk 1 above)
Comment (mikail-khan) [reply app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala]: Thanks, I changed it.

File: app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala (hunk 2 above)
Comment (mikail-khan) [reply app/uk/gov/hmrc/cdsreimbursementclaim/connectors/CcsConnector.scala]: changed.

**Review by gabor-dorcsinecz — APPROVED** [review]

**Pull request merged**